python/tvm/relay/frontend/pytorch.py

```diff
@@ -4470,6 +4558,26 @@ def _run_jit_passes(graph, enable_lower_all_tuples=True):
         torch._C._jit_pass_lower_all_tuples(graph)
 
 
+def _redirect_inplace_output(graph):
```

Comment:
Please give an example of what this pass does, by documenting IR before / after this pass.

Reply:
Ok An example added